stormevents/nhc/track.py

```diff
@@ -1235,6 +1235,52 @@ def correct_ofcl_based_on_carq_n_hollandb(
     :return: dictionary of forecasts for each advisory (aside from best track ``BEST``, which only has one hindcast) with corrected OFCL
     """
 
+    def clamp(n, minn, maxn):
+        return max(min(maxn, n), minn)
+
+    # Regression coefficients for the Rmax forecast
+    # ref: Penny et al. (2023). https://doi.org/10.1175/WAF-D-22-0209.1
+    def get_regression_coefs(fcst_hr, radii_values):
```

Comment:
@WPringle it'd be nice if you could add a simple test for this function, by just providing fcst hour and radii, what do you expect to see. Just a couple of basic cases.

Reply:
added in latest commit